config/constants.yml

```diff
@@ -25,9 +25,8 @@ TERMINUS_TIME_ZONE:     'UTC'
 
 # File Storage
 TERMINUS_CACHE_DIR:     '~/.terminus/cache'
-TERMINUS_CONFIG_DIR:    '~/terminus'
 TERMINUS_LOG_DIR:       '/tmp'
-TERMINUS_PLUGINS_DIR:   '[[ TERMINUS_CONFIG_DIR ]]/plugins'
+TERMINUS_PLUGINS_DIR:   '~/.terminus/plugins'
```

Comment:
Previously, plugins were stored in ~/terminus/plugins. With this PR, they are in ~/.terminus/plugins. While I think it is a good idea to ensure that Terminus 0.x does not behave poorly when presented with Terminus 1.x plugins, I don't really like storing plugins within a hidden directory.

If this was already discussed, and the consensus was a hidden directory, I'll let it go, though; don't feel that strongly about it.